tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,117 @@ $(document).ready(function () {
             $(this).text("- Less Search Filters")
         }
     });
-});
\ No newline at end of file
+});
+
+function pagination() {
+    var numberOfItems = $(" #loop .list-group").length;
+    var limitPerPage = $("#first").val();
+
+    $("#selected-option").change(function() {
+        var o = $(this).find("option:selected");
+        if (o.val() === "All") {
+            limitPerPage = numberOfItems;
+        } else {
+            limitPerPage = o.val()
+        }
+
+        $(".pagination").children("li.to-remove").remove();
```

Comment:
you are removing them? Maybe just hide, then you can easily change between them?

There is a change i'm misunderstanding what you are doing here.

Reply:
So whenever we change the limit, a lot of variables can change like `limitPerPage`, `totalPages`, `currentPage`, which makes it so that we have more or less pages. So I remove all the previous appended html to the pagination class and then append the new buttons onto it.

Reply:
you might have to show me, i'm not fully understanding

Reply:
Changed `to-remove` to `temp-button`